reana_client/utils.py

```diff
@@ -420,8 +396,8 @@ def run_command(cmd, display=True, return_output=False, stderr_output=False):
     """
     now = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
     if display:
-        click.secho("[{0}] ".format(now), bold=True, nl=False, fg="green")
-        click.secho("{0}".format(cmd), bold=True)
+        display_message("[{0}] ".format(now), msg_type="success")
+        display_message("{0}".format(cmd))
```

Comment:
ATM we don't have any `run_command` with `display=True` but I think this can be reverted to `click.secho` and it's basically printing the date and the cmd, not a message _per se_, and we don't need SUCCESS prefix.

Reply:
Reverted to its original version